doc/ref_model/chapters/appendix-a.md

```diff
@@ -70,6 +71,22 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 End of policies.
 
 <a name="A.6"></a>
+## A.6 VNF Design and Implementation Requirements
+This section specifies a set of VNF Design and Implementation Requirements. 
+
+| Ref | Requirmenet | Unit | Definition
+|----|-----|-----|----|
+| vnf.des.001 | The VNF Developer **must** ensure that their software and the resultant VNF image does not contain malicious code (e.g., malware, logic bombs, etc.)  |  | |
+| vnf.des.002 | The VNF Developer **must** ensure that their software and the resultant VNF image does not contain code such as daemons that exposes them to risk |  | |
+| vnf.des.003 | The VNF Developer **must** ensure that their software and resultant VNF image are only created with content and files from trusted sources |  | |
+| vnf.des.004 | The VNF Developer **must** ensure that their software and resultant VNF image are only packaged with files that have been found free of malware and vulnerabilities |  | |
+| vnf.des.005 | The VNF Developer **must** ensure that their software and resultant VNF image do not contain clear text secrets |  | |
```

Comment:
```suggestion
| vnf.des.005 | The VNF Developer **should** ensure that their software and resultant VNF image do not contain clear text secrets |  | |
```

Reply:
@kedmison Why should this not be required? Operators could ask for Vendors to certify to this.

Reply:
Indeed it can be.  But how would it be made a testable requirement at the RC?
Operators can make some of these mandatory during their procurement cycles if so desired.